src/components/Spinner/docs/examples.ts

```diff
@@ -0,0 +1,15 @@
+export const importExample = `import { Spinner } from 'wix-ui-tpa/Spinner';`;
+
+export const regular = `
+<Spinner />
+`;
+
+export const slim = `
+<ExampleWithContextProps mobile={true}>
+  <Spinner type="slim" />
+</ExampleWithContextProps>
+`;
```

Comment:
Again, mobile can be removed:
```suggestion
export const slim = `
<Spinner type="slim" />
`;
```